recipes/libcal/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+cuda_version:
+  - 11
+  - 12
```

Comment:
Let's see if we can make the CI happy (https://github.com/conda-forge/staged-recipes/pull/26580#issuecomment-2153518517):
```suggestion
```